src/docfx/lib/moniker/MonikerSpec.cs

```diff
@@ -6,18 +6,18 @@
 
 namespace Microsoft.Docs.Build
 {
-    internal class Moniker
+    internal class MonikerSpec
     {
         [JsonRequired]
-        public string MonikerName { get; set; }
+        public string Moniker { get; set; }
 
         [JsonRequired]
-        public string ProductName { get; set; }
-
-        [JsonRequired]
-        public int Order { get; set; }
+        public string Product { get; set; }
 
         [JsonExtensionData]
         public JObject ExtensionData { get; } = new JObject();
+
+        public string GetMonikerPropertyValue(string property)
+            => ExtensionData.TryGetValue<JValue>(property, out var v) && v.Value is string str ? str : null;
```

Comment:
where is this used?

Reply:
not used now? should I add it when we use it?

---
In reply to: [230629189](https://github.com/dotnet/docfx/pull/3575#discussion_r230629189) [](ancestors = 230629189)